### DISCUSSION TITLE
q-tree with q-popup-edit

### DISCUSSION THREAD

**Discussion by denprog** [quasarframework/quasar#16582] [General - Components / Directives / etc]
How should I use q-popup-edit in a q-tree? Is it possible? Or, it would be better, edit a string in a q-tree in-place by a double click?
My q-tree is like this:
`        <q-tree :nodes="documents" dense v-model:selected="selectedDocument" ref="documentsRef" node-key="id" label-key="label" 
            :filter="documentsFilter" @update:selected="onDocumentSelected" default-expand-all>
        </q-tree>
`
I can't attach a popup edit using the docs...

**Comment by metalsadman** [comment]
use the header slot, then you can put your qpopedit there.
